gui/quantum.lua

```diff
@@ -11,262 +11,212 @@ local quickfort = reqscript('quickfort')
 local quickfort_command = reqscript('internal/quickfort/command')
 local quickfort_orders = reqscript('internal/quickfort/orders')
 
-QuantumUI = defclass(QuantumUI, guidm.MenuOverlay)
+
+-- UI Layout
+QuantumUI = defclass(QuantumUI, gui.ZScreen)
 QuantumUI.ATTRS {
-    frame_inset=1,
     focus_path='quantum',
-    sidebar_mode=df.ui_sidebar_mode.LookAround,
+	defocusable=true
```

Comment:
`defocusable` is true by default, so you can remove that one. I also suggest you follow the example of `gui/blueprint` and set
```lua
pass_movement_keys=true,
pass_mouse_clicks=false,
```